src/node_crypto.cc

```diff
@@ -990,7 +1001,7 @@ void SecureContext::LoadPKCS12(const FunctionCallbackInfo<Value>& args) {
   if (cert != nullptr)
     X509_free(cert);
   if (extra_certs != nullptr)
-    sk_X509_free(extra_certs);
+    sk_X509_pop_free(extra_certs, X509_free);
```

Comment:
might make sense to put as two commits for back-porting purposes... I don't see why they couldn't both come in on this PR though